cms/templates/studio_xblock_wrapper.html

```diff
@@ -158,7 +158,15 @@
 
 % if show_preview:
     % if is_root or not xblock_url:
-        <article class="xblock-render">
+        % if not is_root:
```

Comment:
Could simplify this a bit (lines 161-169). You could either do this:

````
% if not is_root and language:
    <article class="xblock-render" lang="${language}">
% else:
    <article class="xblock-render">
% endif
````

or

````
<article class="xblock-render"
% if not is_root and language:
    lang="${language}"
% endif
>
````

But can you also explain why we only want to add the lang attribute if is_root is False? I'm not suggesting it is wrong, I just don't understand.